src/main/java/no/finn/unleash/event/UnleashSubscriber.java

```diff
@@ -0,0 +1,21 @@
+package no.finn.unleash.event;
+
+import no.finn.unleash.UnleashException;
+import no.finn.unleash.metric.ClientMetrics;
+import no.finn.unleash.metric.ClientRegistration;
+import no.finn.unleash.repository.FeatureToggleResponse;
+import no.finn.unleash.repository.ToggleBackupHandlerFile;
+import no.finn.unleash.repository.ToggleCollection;
+
+public interface UnleashSubscriber {
+
+    default void on(UnleashEvent unleashEvent) { }
+    default void onError(UnleashException unleashException) { }
+    default void toggleEvaluated(ToggleEvaluated toggleEvaluated) { }
+    default void togglesFetched(FeatureToggleResponse toggleResponse) { }
+    default void clientMetrics(ClientMetrics clientMetrics) { }
+    default void clientRegistered(ClientRegistration clientRegistration) { }
+    default void togglesBackedUp(ToggleCollection toggleCollection) { }
+    default void toggleBackupRestored(ToggleCollection toggleCollection) { }
+
```

Comment:
I was thinking that there is one event missing here: `onReady` which should be triggered as soon as the client has successfully fetched from the server.

Reply:
Like this?
https://github.com/Unleash/unleash-client-java/pull/67/files#diff-7fc449c201ef2331c2e8474f2d9ace6eR68
https://github.com/Unleash/unleash-client-java/pull/67/files#diff-c1feded3c37b2e4f9d387af14ea0bf3cR17